src/App.js

```diff
@@ -3,6 +3,8 @@ import {Home} from './pages/home/home'
 import {Products} from './pages/products/products'
 import {Cart} from './pages/cart/cart'
 import { Login } from "./pages/login/login";
+import { SignUp } from "./pages/sign-up/sign-up";
+import { Wishlist } from "./pages/wishlist/wishlist";
 import {Routes,Route} from 'react-router-dom'
```

Comment:
you can use single index file to import the pages

Reply:
ok will do it